blazor/spreadsheet/undo-redo.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Undo Redo in Blazor Spreadsheet Component"
+component: "Spreadsheet"
+description: "Learn how to use the undo and redo functionality in the Syncfusion Blazor Spreadsheet component."
+platform: Blazor
+control: Spreadsheet
+documentation: ug
+---
+
+# Undo Redo in Spreadsheet
+
+<b>Undo</b> option helps you to undone the last action performed and <b>Redo </b>option helps you to do the same action which is reverted in the Spreadsheet. 
+
+
+## Undo
+
+It reverses the last action you performed with Spreadsheet. <b>Undo</b> can be done by any of the following ways:
+
+* Select the undo item from HOME tab in Ribbon toolbar.
+
+* Use <kbd>Ctrl</kbd> + <kbd>Z</kbd> keyboard shortcut to perform the undo.
+
+## Redo
+
+It reverses the last undo action you performed with Spreadsheet. <b>Redo</b> can be done by any of the following ways:
+
+* Select the redo item from HOME tab in Ribbon toolbar.
+
+* Use <kbd>Ctrl</kbd> + <kbd>Y</kbd> keyboard shortcut to perform the redo.
+
+
```

Comment:
Attach a screenshot for Redo button
like as follows,
![image](https://github.com/user-attachments/assets/16111d7d-fdd3-4893-b1fa-cee556c091c1)